GVRf/Framework/framework/src/main/java/org/gearvrf/x3d/TextureValues.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (C) 2006 The Android Open Source Project
```

Comment:
TextureValues.java may be removed eventually; replaced with a single value array list shared when parsing Texture Coordinates, indexedFaceSets, Normals, etc.  Still, the copyright information has been replaced on this and other code.